src/repo/xdb_repo_queryable.erl

```diff
@@ -100,6 +101,11 @@ get_by_or_raise(Repo, Adapter, Queryable, Clauses) ->
 get_by_or_raise(Repo, Adapter, Queryable, Clauses, Opts) when is_atom(Queryable) ->
   one_or_raise(Repo, Adapter, Queryable, Clauses, Opts).
 
+%% @private
+all_by(Repo, Adapter, Queryable, Clauses) ->
```

Comment:
I don't understand the point of this function, you can just call:

```erlang
all(Repo, Adapter, xdb_query:from(Queryable, [{where, Clauses}]))
```

The `all` function is generic enough, if you want to customize it, it should be done in your repo, that's the idea.

Reply:
@cabol it's a handy shortcut, the same as we have `get` and `get_by`